tests/test_end_to_end.py

```diff
@@ -27,64 +27,64 @@ def setUp(self):
         self._wtTestFixture.setUp()
         self.setUpServer(backend.Backend(self._wtTestFixture.dataDir,
                                          variants.WormtableVariantSet))
+        self._expectedVariantSetIds = ['example_1', 'example_2',
+                                       'example_3', 'example_4']
 
     def tearDown(self):
         self._wtTestFixture.tearDown()
 
-    def sendJSONPostRequest(self, path, data):
-        return self.app.post(path,
-                             headers={'Content-type': 'application/json'},
-                             data=data)
+    def sendJsonPostRequest(self, path, request):
+        """send json post, assert status code to be 200,
+        and return response data"""
+        response = self.app.post(path,
+                                 headers={'Content-type': 'application/json'},
+                                 data=request.toJsonString())
+        self.assertEqual(200, response.status_code)
+        if path == "/callsets/search":
+            responseData = protocol.GASearchCallSetsResponse.\
```

Comment:
Again, I'd rather see this formatted as:
```
responseData = protocol. GASearchCallSetsResponse.fromJsonString(
    response.data)
```
(unless it can't fit on that line, in which case what you had was fine)